packages/ui-plugin-sheets/src/controllers/contextmenu/contextmenu.controller.ts

```diff
@@ -58,7 +69,15 @@ export class SheetContextMenuController extends Disposable {
         const spreadsheetPointerDownObserver = objects.spreadsheet.onPointerDownObserver;
         const spreadsheetObserver = spreadsheetPointerDownObserver.add((event) => {
             if (event.button === 2) {
-                this._contextMenuService.triggerContextMenu(event, MenuPosition.CONTEXT_MENU);
+                const selections = this._selectionManagerService.getSelections();
```

Comment:
The commented code above should be removed.

Reply:
history code, i will remove it.